bitcoin-magazine/bitcoin-2024/making-bitcoin-more-private-with-cisa.md

```diff
@@ -184,110 +187,119 @@ So that's how I would encourage you to think about it from the start.
-It's interesting because I was in the room in Hong Kong when Peter Wille announced that we were changing the flagging system so that we could do like 32 soft forks in parallel.
+It's interesting because I was in the room in Hong Kong when Pieter Wuille announced that we were changing the flagging system so that we could do like 32 soft forks in parallel.
 And of course, we have not yet taken advantage of that.
 
-Speaker 2: 00:16:33
+Craig Raw: 00:16:33
 
 Yeah, so we're in a very different space from that now.
-I think something to bear in mind, if you're thinking about it from that point of view, is that, and I heard Brandon Black actually, who was here earlier today talking about Opcad, is that I think Caesar is probably the most low-risk soft fork that we could imagine.
-And I'm talking here about the well-understood half signature aggregate form of it.
+I think something to bear in mind, if you're thinking about it from that point of view, is that, and I heard Brandon Black actually, who was here earlier today talking about Opcad, is that I think CISA is probably the most low-risk soft fork that we could imagine.
+And I'm talking here about the well-understood half-signature aggregate form of it.
 You know, it's really from a security model point of view, the cryptographers tell us that there is no risk effectively or very low risk to doing this.
 It really takes very well understood properties of Schnorr, which is really just adding these signatures up and uses that.
 
 ## The Limited Potential of Signature Compression
 
-Speaker 2: 00:17:18
 
 So in terms of risk of doing the soft fork it's not really enabling other things that Bitcoin can do.
 It's not going to be some kind of ungranted, it's really just compressing.
 And if you think about it, compression is a fairly bounded area.
 You're not going to suddenly have people developing all kinds of perhaps unwanted things on top of Bitcoin just because you have compressed the signature size down.
 So maybe it opens up the space for other soft forks if we do a soft fork which has such a low risk to it that it actually gets across the line.
 
-Speaker 3: 00:17:57
+Jameson Lopp: 00:17:57
 
 I think it's also worth noting that this is by no means the only signature aggregation scheme that's even happening in Bitcoin, right?
-Is that like with the taproot soft fork a few years ago, we did get the theoretical ability to do Schnorr signatures.
+Is that like with the Taproot soft fork a few years ago, we did get the theoretical ability to do Schnorr signatures.
 But that has been an ongoing process, right?
-So as far as I'm aware off the top of my head, there's a pretty limited amount of adoption of Schnorr so far because I think Music, Music 2, is the main production-ready standard that people are using.
+So as far as I'm aware off the top of my head, there's a pretty limited amount of adoption of Schnorr so far because I think Musig 2, is the main production-ready standard that people are using.
```

Comment:
```suggestion
So as far as I'm aware off the top of my head, there's a pretty limited amount of adoption of Schnorr so far because I think MuSig2, is the main production-ready standard that people are using.
```

Reply:
I was to change these just forgot after edits.